### PR TITLE
Inelastic neutron scattering (INS) panel in results

### PR DESCRIPTION
**Pull request by mikibonacci**
This will add the INS panel in the results. 
Also, in the repository, I put a notebook (aiidalab_qe_vibroscopy/utils/euphonic/Detached_app.ipynb) which basically allows to plot INS data if we have phonopy.yaml and force_constanst.hdf5 files. the python widgets are the same, so the code to be maintained is only one.

### CODE REVIEW

**Comment by mikibonacci** [comment]
Probably, instead of merge, we can do a Squash, and I will take care of resuming the main things I committed

File: phonopy_install.py [hunk 1]
```diff
@@ -4,6 +4,14 @@
 from aiida import load_profile
 import shutil
 
+"""
+Automatic installation of the phonopy code.
+
+(1) if not already installed: pip install phonopy
+(2) in not already there: ln -s <phonopy path> /opt/conda/bin/phonopy (try which phonopy to see the phonopy path)
```
Comment (AndresOrtegaGuerrero) [inline phonopy_install.py:11]: `If it doesn't already exist, create a symbolic link for Phonopy`

**Review by AndresOrtegaGuerrero — APPROVED** [review]
LGTM! , I just left few suggestions!

**Comment by AndresOrtegaGuerrero** [comment]
should we merge it ?

**Comment by mikibonacci** [comment]
I just need to complete your suggestions (btw, thanks) and then I will merge it!

**Comment by mikibonacci** [comment]
@AndresOrtegaGuerrero , I just merged the PR, adding also your suggestions. Thanks a lot!

**Pull request merged**